wordless/theme_builder/vanilla_theme/Procfile

```diff
@@ -1,3 +1,3 @@
 wp: wp server --host=0.0.0.0
-webpack: npx webpack --debug --watch --progress --color --env.NODE_ENV=development
```

Comment:
Need to understand if this can be a problem.

with webpack5 and the new webpack-cli we don't have some flags 
from the documentation @v.4

--debug --> Switch loaders to debug mode
--bail --> Abort the compilation on first error
--p --optimize-minimize --define process.env.NODE_ENV="production"